<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisQC</class>
 <widget class="QWidget" name="analysisQC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>551</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QC Stats (all spots and genes)</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Total Spots</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="meanTranscripts">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The median of transcripts per spot</string>
       </property>
       <property name="statusTip">
        <string>The median of transcripts per spot</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Max genes</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="maxTranscripts">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The maximum number of transcripts in a spot</string>
       </property>
       <property name="statusTip">
        <string>The maximum number of transcripts in a spot</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Mean(spot)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="meanGenes">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The median os genes per spot</string>
       </property>
       <property name="statusTip">
        <string>The median os genes per spot</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="totalSpots">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The total number of spots</string>
       </property>
       <property name="statusTip">
        <string>The total number of spots</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Transcripts</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="totalGenes">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The total number of genes</string>
       </property>
       <property name="statusTip">
        <string>The total number of genes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Std(spot)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Std(spot)</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Max transcripts</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="stdGenes">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The standard deviation os genes per spot</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="totalTranscripts">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>75</horstretch>
         <verstretch>21</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The total number of transcripts</string>
       </property>
       <property name="statusTip">
        <string>The total number of transcripts</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mean(spot)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="stdTranscripts">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="toolTip">
        <string>The standard deviation of transcripts per spot</string>
       </property>
       <property name="statusTip">
        <string>The standard deviation of transcripts per spot</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>General</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="maxGenes">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>The maximum number of genes in a spot</string>
       </property>
       <property name="statusTip">
        <string>The maximum number of genes in a spot</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QPushButton" name="exportGenes">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Eport the genes' histogram to a file</string>
       </property>
       <property name="statusTip">
        <string>Eport the genes' histogram to a file</string>
       </property>
       <property name="text">
        <string>Export Genes</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="exportTranscripts">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Eport the transcripts' histogram to a file</string>
       </property>
       <property name="statusTip">
        <string>Eport the transcritps' histogram to a file</string>
       </property>
       <property name="text">
        <string>Export Transcripts</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Genes</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="ChartView" name="genesPlot">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ChartView" name="transcriptsPlot">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>analysis/ChartView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
